app/views/layouts/session.html.erb

```diff
@@ -38,7 +38,7 @@
       <div id="inner" class="wrapper">
         <!-- BEGIN main -->
         <div id="main" class="<%= controller.controller_name + '-' + controller.action_name %> system session region" role="main">
-          <%= flash_div :error, :caution, :notice %>
+          <%= flash_div :error, :caution, :notice, :alert %>
```

Comment:
I think we need to add `.alert` to all the places `.error` appears in 22-system-messages.css, except `span.error`. Otherwise it'll be unstyled, as we saw with the "You need to sign in or sign up before continuing" message on dev.

Reply:
I'm pretty sure we don't need in in `div.error` either so I'm not going to add it at the moment. If we run into problems, 🐟 slap me later.